QueueIntegrations/Prognetics.Saga.Queue.RabbitMQ/ChannelSetup/RabbitMqQueue.cs

```diff
@@ -4,7 +4,7 @@ public class RabbitMQQueue
 {
     public required string Name { get; init; }
 
-    public string Exchange { get; init; } = string.Empty;
+    public string? Exchange { get; init; }
```

Comment:
Why an explicit nullable here?

Reply:
Then its clear that the user is able to not define the exchange and use the default one. A nullable property is more explicit.